### ISSUE TITLE
WinForms and BusinessBindingListBase

### ISSUE THREAD

**Issue by Chicagoan2016** [MarimerLLC/cslaforum#380]
Are classes like BusinessBindingListBase and DynamicBindingListbase still recommended while developing
Winforms applications? I couldn't find references to  them in Project tracker code.

Regards

**Comment by jonnybee** [comment]
Yes, they are required in order to fully use WinForms data binding.
